src/avi/render.clj

```diff
@@ -29,7 +29,10 @@
                          (color/make :blue :black))
             line-text (or document-line "~")]
         (fill-rendition-line! rendition (+ i from-line) [line-color line-text])))
-    (fill-rendition-line! rendition to-line [(color/make :black :white) (or (:name document) "[No Name]")])))
+    (let [file-name (or (:name document) "[No Name]")
+          [i j] (:point (e/edit-context editor))
+          msg-txt (str file-name "\t[" (inc i) "," (inc j) "]" )]
```

Comment:
Changed from tab to 3 spaces. Need to work on limiting filename charecters on the status line.
Also another work TODO is to not change point in the panes that are not active while working on multiple panes with same buffer ":sp"